### ISSUE TITLE
ES6 support

### ISSUE THREAD

**Issue by pawelbaranski** [dsheiko/jscodesniffer#29]
Hi

Is ES6 going to be supported ? esprima version you are using only supports 5.1, but the newest version has ES6 support. Is there a reason why it is no used yet ?

**Comment by dsheiko** [comment]
Yeah, I was checking some time ago, Esprima is fine with ES6. I will take care soon
